### DISCUSSION TITLE
Clickhouse Dialect for hibernate 6

### DISCUSSION THREAD

**Discussion by sinfull1** [hibernate/hibernate-orm#6855] [Polls]
Hi,

Use of clickhouse is hot in analyltics

I am using the Clickhouse database for capture/bulk insert of events and running analysis queries on it.
Majority of the dialect is based out of MYSQL db, but has few new JDBC constructs

Would it make sense create an issue to write the dialect for Clickhouse DB ?

For my work, I have written a dialect that works for my use cases, could share it as initial starter

Regards,
Siddharth

**Comment by beikov** [comment]
Maybe ask the Clickhouse folks if they want to contribute and maintain the dialect, as well as regularly test it. Either way, you can contribute that dialect into the `hibernate-community-dialects` module if you want and maintain it yourself.

**Comment by sinfull1** [comment]
Hello Christian, could you please provide me permission to create pull request

**Reply by Sanne** [reply]
Hello! Anyone can create pull requests, you don't need special permissions: just go ahead and create a pull request.

**Reply by sinfull1** [reply]
Ok, I am unable to push a branch, would you know which permission is required for that.. ?

**Reply by Sanne** [reply]
You need to push to a branch on your own fork, then send a pull request from there.
